pkg/controllers/cloud/account_poller.go

```diff
@@ -76,11 +87,25 @@ func (p *accountPoller) doAccountPoller() {
 	if e != nil {
 		p.log.Info("failed to update account status", "account", p.namespacedName, "err", e)
 	}
-	virtualMachines := p.getComputeResources(cloudInterface)
 
-	e = p.doVirtualMachineOperations(virtualMachines)
+	vpcMap, e := cloudInterface.GetVpcInventory(p.namespacedName)
```

Comment:
Its little confusing to read the code.. you are calling GetVPCInventory here and in line 97 you are calling BuildVPCInventory. If you already got a VPC, why build in line 97. I know these are different Caches, but from the code is not clear.. Please update the function names

Reply:
Agree, renamed buildVpcInventory() function to buildVpcCache(). It think it makes more sense now as the 1st one gets vpc inventory and the 2nd function build vpc cache out of it.